docs/product-release-notes/src/main/asciidoc/con-bxms-rn-tech-preview.adoc

```diff
@@ -0,0 +1,34 @@
+
+[[bxms_rn_tech_preview]]
+= Technical Preview
+
+This section lists features that are in technical preview in {PRODUCT} {PRODUCT_VERSION} Limited Availability. Where they are listed in the documentation these features also have a note similar to the following:
+
+[IMPORTANT]
+====
+This feature a Technology Preview feature only. Technology Preview features
+are not supported with Red Hat production service level agreements (SLAs), might
+not be functionally complete, and Red Hat does not recommend to use them for
+production. These features provide early access to upcoming product features,
+enabling customers to test functionality and provide feedback during the
+development process.
+For more information on Red Hat Technology Preview features support scope, 
+see https://access.redhat.com/support/offerings/techpreview/
+====
+
+* Deployments perspective 
++
+This perspective is disabled by default. Perform the following steps to enable it:
++
+. Go to *Settings* -> *Roles*.
+. Select the *admin* role.
+. In the Permissions section, click the arrow next to Perspectives.
+. Click the trash icon next to Deployments. The DevOps -> *Deployments* link is now available on the Home page.
+
+* Guided Decision Trees
+* Guided Score Cards
+* XLS Score Cards
+* New jBPM Designer
+* Provisioning perspective
```

Comment:
Afaik this is the same as the deployments perspective, suggest to drop this one.